boot/env.ml

```diff
@@ -58,7 +58,7 @@ let fail_lib lib =
   let open Printf in
   eprintf "File not found: %s\n" lib;
   eprintf "The path for Coq libraries is wrong.\n";
-  eprintf "Coq libraries are shipped in the coq-stdlib package.\n";
+  eprintf "Coq prelude is shipped in the rocq-init package.\n";
   eprintf "Please check the COQLIB env variable or the -coqlib option.\n";
```

Comment:
```suggestion
  eprintf "Please check the ROCQLIB env variable or the -rocqlib option.\n";
```

If you're changing those, too.  But couldn't these cause the user's build env to fail?  If we go that route, we should give clear explanation of how to migrate in the release notes.

Reply:
`rocq-init` is a new package, so that's the reason for using the Rocq name from the beginning, but for everything else, it's orthogonal to the stdlib split and should not happen in the same PR. There will be work on changing this kind of stuff, but that's not the place to discuss about this.